noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/component/text/NoteTextFields.kt

```diff
@@ -36,7 +36,7 @@ fun NoteTitleField(
         value = value,
         label = "Title",
         onTextChange = onTextChange,
-        textStyle = MaterialTheme.typography.h6,
+        textStyle = MaterialTheme.typography.h4.copy(fontWeight = FontWeight.Bold),
```

Comment:
I guess `h6` was good. Doesn't it (title) look bigger than it should be?

Reply:
hmm yes 🤔